t/prometheus-metrics.t

```diff
@@ -275,7 +315,144 @@ qr/upstream_response_time_seconds(.|\n)*upstream_response_time_seconds_bucket\{l
 [
 "",
-qr/total_response_time_seconds(.|\n)*total_response_time_seconds_bucket\{le=".*"\} 1/
+qr/total_response_time_seconds(.|\n)*total_response_time_seconds_bucket\{service="",le=".*"\} 1/
+]
+--- no_error_log
+[error]
+
+
+
+=== TEST 6: extended metrics show services id
+--- env eval
+("APICAST_EXTENDED_METRICS", "true")
+--- configuration
+{
+  "services": [
+    {
+      "id": 42,
+      "proxy": {
+        "policy_chain": [
+          {
+            "name": "apicast.policy.upstream",
+            "configuration": {
+              "rules": [
+                {
+                  "regex": "/",
+                  "url": "http://test:$TEST_NGINX_SERVER_PORT"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    }
+  ]
+}
+--- upstream
+  location / {
+     content_by_lua_block {
+       ngx.exit(200);
+     }
+  }
+
+--- request eval
+["GET /", "GET /metrics"]
+--- more_headers eval
+["", "Host: metrics"]
+--- error_code eval
+[ 200, 200 ]
+--- expected_response_body_like_multiple eval
+[
+"",
+[
+    qr/total_response_time_seconds(.|\n)*total_response_time_seconds_bucket\{service="42",le=".*"\} 1/,
+    qr/upstream_response_time_seconds(.|\n)*upstream_response_time_seconds_bucket\{service="42",le=".*"\} 1(.|\n)*upstream_status\{status="200",service="42"\} 1/
+]
 ]
 --- no_error_log
 [error]
+
+=== TEST 7: extended metrics with multiple services
+--- env eval
+("APICAST_EXTENDED_METRICS", "true")
+--- configuration
+{
+  "services": [
+    {
+      "id": 42,
+      "proxy": {
+        "hosts": [
+          "one"
+        ],
+        "policy_chain": [
+          {
+            "name": "apicast.policy.upstream",
+            "configuration": {
+              "rules": [
+                {
+                  "regex": "/",
+                  "url": "http://test:$TEST_NGINX_SERVER_PORT"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    },
+    {
+      "id": 21,
+      "proxy": {
+        "hosts": [
+          "two"
+        ],
+        "policy_chain": [
+          {
+            "name": "apicast.policy.upstream",
+            "configuration": {
+              "rules": [
+                {
+                  "regex": "/",
+                  "url": "http://test:$TEST_NGINX_SERVER_PORT/two"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    }
+  ]
+}
+--- upstream
+  location / {
+     content_by_lua_block {
+       ngx.exit(200);
+     }
+  }
+
+  location /two {
+     content_by_lua_block {
+       ngx.exit(200);
+     }
+  }
+--- request eval
+["GET /", "GET /two", "GET /metrics"]
+--- more_headers eval
+["Host: one", "Host: two", "Host: metrics"]
+--- error_code eval
+[ 200, 200, 200 ]
+--- expected_response_body_like_multiple eval
+[
+"",
+"",
+[
+qr/total_response_time_seconds_bucket\{service="42",le=".*"\} 1/,
+qr/upstream_response_time_seconds_bucket\{service="42",le=".*"\} 1/,
+qr/upstream_status\{status="200",service="42"\}/,
```

Comment:
I think it would be good to check the counter as well like in the 2 cases above. (`1/`)
Same for service 21.